benchmarks/ddtrace_run/config.yaml

```
@@ -0,0 +1,46 @@
+ddtrace_run: &defaults
+  with_ddtrace_run: "true"
+  with_httpretty: "false"
+  runtime_metrics_enabled: "false"
+  telemetry_enabled: "false"
+  with_trace: "false"
+  profiling_enabled: "false"
+  appsec_enabled: "false"
```

I would simplify this a bit and make this "baseline" include the http checks as this is used when looking at each service.

```suggestion
auto_baseline: &defaults
  ddtrace_run: "true"
  http: "true"
  runtimemetrics: "false"
  telemetry: "false"
  tracing: "false"
  profiling: "false"
  appsec: "false"
```